### PR TITLE
JWWEB-729: Hotfix Select Scenario buttons

### PR DESCRIPTION
**Pull request by stcherenkov**

### CODE REVIEW

File: src/pages/WalletsStart/components/NewWalletButtons/NewWalletButtons.js [hunk 1]
```diff
@@ -31,28 +34,27 @@ export class NewWalletButtons extends PureComponent<Props> {
...
+        </JRaisedButton>
+        <div className={newWalletButtonsStyle.text}>
           {t`Create your own wallet to manage your digital assets`}
-        </span>
-        <JLink
+        </div>
+        <JRaisedButton
           onClick={this.handleClickImport}
-          className={newWalletButtonsStyle.import}
-          href='/wallets/import'
```
Comment (IViol) [inline src/pages/WalletsStart/components/NewWalletButtons/NewWalletButtons.js]: why did you remove `href`s?

Comment (IViol) [reply src/pages/WalletsStart/components/NewWalletButtons/NewWalletButtons.js]: JLink buttons could pass user to separate pages, when he wants to open them in new tab

Comment (stcherenkov) [reply src/pages/WalletsStart/components/NewWalletButtons/NewWalletButtons.js]: Reverted to the old behaviour!

**Comment by Creepypoke** [comment]
@stcherenkov Resolve conflicts, please
